spec/card_spec.rb

```diff
@@ -44,5 +45,11 @@
         expect(card.right_translation?("Hello")).to eq(false)
       end
     end
+
+    describe "user" do
+      it "validates that the card belongs to a user" do
+        expect(card.user.name).to eq("kenn")
+      end
+    end
```

Comment:
@davetoxa do you have a guideline of what methods to test? kind of struggling to decide what i should test vs what i shouldn't test